docs/webpack.md

```diff
@@ -0,0 +1,103 @@
+# Webpack usage examples
+
+## Install dependencies
+
+```bash
+$ npm install --save-dev webpack webpack-cli yargs
+```
+
+Additional dependencies for typescript users:
+
+```bash
+$ npm install --save-dev ts-loader typescript @types/yargs
+```
+
+## Sample program
+
+Create `src/index.js`:
+```js
+const yargs = require('yargs')
+
+console.log(yargs.parse())
+```
+
+Or for typescript users, `src/index.ts`:
+```ts
+import * as yargs from 'yargs';
```

Comment:
I think we're going to want to update the TypeScript examples themselves to use this approach as well, we swapped to the following approach:

https://github.com/yargs/yargs/commit/b1b156a3eb4ddd6803fbbd56c611a77919293000

But it seems to be causing trouble for folks, see: 

https://github.com/yargs/yargs/issues/1455

I think we actually addressed #1439 elsewhere, by removing the deprecation warning.

Reply:
#1455 is on my TODO list, once I'm done with #1452